apache/trafficcontrol
# Use the Python Postinstall implementation by default

## What does this PR (Pull Request) do?

- Makes Postinstall use the Python backend instead of the old Perl one
- Deprecates the Perl version of the Postinstall script
- CDN in  Box now configures Traffic Ops using the Python rewrite of the TO Postinstall script (fixes #5680).
- Adds a GitHub Actions workflow to run the Postinstall unit tests

- [x] This PR fixes #3437 by documenting JSON input to the Postinstall script. We do not need to remove `input.json` because if it ever were to become out-of-date, CDN in a Box would break.

## Which Traffic Control components are affected by this PR?


- CDN in a Box
- Documentation
- Traffic Ops Postinstall script
- CI tests

## What is the best way to verify this PR?

- Run `postinstall.test.sh`
- Verify that the CDN in a Box Delivery Service can be reached

## The following criteria are ALL met by this PR




## Additional Information
